tensorflow/lite/micro/kernels/quantize.cc

```diff
@@ -43,8 +43,9 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   TF_LITE_ENSURE(context, affine_quantization->scale);
   TF_LITE_ENSURE(context, affine_quantization->scale->size == 1);
 
-  TF_LITE_ENSURE(context,
-                 input->type == kTfLiteFloat32 || input->type == kTfLiteInt16);
+  TF_LITE_ENSURE(context, input->type == kTfLiteFloat32 ||
+                              input->type == kTfLiteInt16 ||
```

Comment:
The indentation looks a bit off here, running clang-format will take case of that, as described in this link:
https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#c-coding-style

Reply:
I ran it through `clang-format --style=google` and that is the way it came out. clang-format version 6.0.0